recipients_app/lib/services/authService.dart

```diff
@@ -24,15 +24,15 @@ class AuthService {
     }
     FirebaseAuth.instance.verifyPhoneNumber(
         phoneNumber: phoneNumber,
-        timeout: Duration(seconds: 60),
+        timeout: const Duration(seconds: 60),
         verificationCompleted:
             verificationCompleted ?? (phoneAuthCredential) {},
         verificationFailed: (e) {
-          print("------- ${e.message}");
+          log("------- ${e.message}");
         },
         codeSent: codeSent ?? (verificationId, forceResendCode) {},
         codeAutoRetrievalTimeout: (e) {
-          print("autoretrievel timeout");
+          log("autoretrievel timeout");
```

Comment:
Will it be stripped in release logs with `log` instead of `print`?

Reply:
print should never be used in production since everyone could see the prints we're making. Log is just visible in debug mode. debugPrint is also fine.

https://dart-lang.github.io/linter/lints/avoid_print.html